nilearn/datasets/tests/test_func.py

```diff
@@ -362,3 +363,37 @@ def test_fetch_mixed_gambles():
         assert_equal(mgambles["zmaps"][0], os.path.join(datasetdir, "zmaps",
                                                         "sub001_zmaps.nii.gz"))
         assert_equal(len(mgambles["zmaps"]), n_subjects)
+
+
+@with_setup(tst.setup_tmpdata, tst.teardown_tmpdata)
+def test_fetch_megatrawls_netmats():
+    # smoke test to see that files are fetched and read properly
+    # since we are loading information present in it and returning
+    # the same
+    dimensionality = [25, 100, 200, 300]
```

Comment:
Aren't you missing some cases here. Dimensinoalities appear to be ``[25, 50, 100, 200, 300]``